crossplane/kcl.mod

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "crossplane"
-version = "1.15.2"
+version = "1.15.2.1"
```

Comment:
Could you pls bump the version 1.15.3? Make it a valid sematic version. Thank you.